lib/hammer_cli_foreman_remote_execution/interface_extensions.rb

```diff
@@ -1,13 +1,35 @@
+# frozen_string_literal: true
+
 require 'hammer_cli_foreman/interface'
 
 module HammerCLIForemanRemoteExecution
-  class InterfaceExtensions < ::HammerCLI::CommandExtensions
+  class InterfaceExtensionsInfo < ::HammerCLI::CommandExtensions
     output do |definition|
       definition.append do
         field :execution, _('Execution'), Fields::Boolean
       end
     end
   end
 
-  ::HammerCLIForeman::Interface::InfoCommand.extend_with(InterfaceExtensions.new)
+  module InterfaceExtensionsList
+    def self.included(base)
+      base.class_eval do
+        def extend_data(nic)
+          type = HammerCLIForeman::Interface.format_type(nic)
+          if nic['execution']
+            if nic['primary'] || nic['provision']
+              type[')'] = ', '+_('execution') +')'
```

Comment:
Here we could assume that if the interface has primary or provision flag, then the `)` character will be the last one in type and it would be better to append to a substring of the original type than perform match-and-replace.